src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  sides: number[];
```

Comment:
I think better write this without spaces
```suggestion
  shape = 'triangle';
  color: string;
  sides: number[];
```